### ISSUE TITLE
Create new issue templates for adding/removing single volunteers on a project page with instructions

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#7478]
### Overview
We need a template for adding and removing single leaders, so that we can easily produce quick leadership change issues

### Details
These templates and instructions are meant to be temporary until the wiki instruction process and templates is finished.

### Action Items
- Create new templates based off of a larger issue (that Jessica is drafting) in a comment below
   - [x] Adding a new member
   - [x] Removing a new member
- Write up instructions for using these templates
   - [x] Adding a new member
   - [x] Removing a new member

### Resources/Instructions
- https://github.com/hackforla/website/issues/5630
- Instructions for 
   - [Adding a leader](https://github.com/hackforla/website/issues/7478#issuecomment-2356611920)
   - [Removing a leader](https://github.com/hackforla/website/issues/7478#issuecomment-2356626767)

**Comment by ExperimentsInHonesty** [comment]
# New template issue draft to add a single project member.
## Template Name
what appears on the new issue template page
```
Project Profile leader updates - Add
```

## About
what appears on the new issue template page
```
This template is for website team only to ADD a single volunteer 
```

## Body
````
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] In your IDE, open the `_projects/[Insert filename].md` file.
- [ ] Observe the existing syntax of the front matter block [^1] in the file.
- [ ] Find the `leadership` variable and add the following profile. 
```
  - name: [Insert leadership member's name]
    github-handle: [Insert GitHub handle]
    role: [Insert role]
    links:
      slack: https://hackforla.slack.com/team/[Insert Slack member ID]
      github: https://github.com/[Insert GitHub handle]
    picture: https://avatars.githubusercontent.com/[Insert GitHub handle]
```
- [ ] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [ ] [Insert name of project] page [^2]
- [ ] Once your pull request is merged, go to the initiating ER [^3]
  - [ ] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move initiating ER [^3] to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: [Insert project specific page URL here]
[^3]: initiating ER:  #[Insert ER number]
````
## Title
title of the issue that is created
```
Update Project Profile: [Project Name] Add [Name of Volunteer]
```

**Comment by ExperimentsInHonesty** [comment]
# Instructions for filling out the Add a leader template

1. Create a new issue using the [add new leader to project](https://github.com/hackforla/website/issues/new?assignees=&labels=good+first+issue%2C+P-Feature%3A+Project+Info+and+Page%2C+role%3A+back+end%2FdevOps%2C+role%3A+front+end%2C+size%3A+0.25pt%2C+time+sensitive&projects=&template=project-profile-leader-updates---add.md&title=Update+Project+Profile%3A+%5BProject+Name%5D+Add+%5BName+of+Volunteer%5D) template
2. Customize the title with name of project and person to add
3. Customize the body
    1.  In the ER, go to the value under the line "Project Card URL"
       1. Copy the file name 
       2. On the line that starts "- [ ] In your IDE…" replace brackets with the copy (the text we are expecting is a file name with the extension .md)
    3. In the copy block below the line that starts, "Find the `leadership` variable and add the following profile."  add all the information from the ER for the person you are adding.
    4. On the checkbox line below, "- [ ] Verify the changes by viewing..." replace with name of project
      1. to Resource 3, find the project on the https://www.hackforla.org/projects/ page
          1. click on its link
          3. copy link
          5. replace brackets on resource 3 with the URL
6. Under "Relationships", add the originating ER as a parent
7.  Choose `Update comment` button
8. Add milestone `Onboarding flow`
9. Review, and if ready, add label```
Ready for Prioritization
```
9. Add the URL to the ER
   1. if this is the first add leader issue to be created, paste this into a comment and add the URL of the issue you just created.
```
### Adding Leader issues
-

### Complete
- [ ] Check this when all the Add leader issues have been made
```
   2. if there are already the block above in the issue, just add your new issue URL to the list. 
      1. Checking off the box, if its the last one that needed to be created.

**Comment by ExperimentsInHonesty** [comment]
# New template issue draft to remove a single project member.

## Template Name
what appears on the new issue template page
```
Project Profile leader updates - Remove
```

## About
what appears on the new issue template page
```
This template is for website team only to REMOVE a single volunteer 
```

## Body

````
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] In your IDE, open the `_projects/[Insert filename].md` file.
- [ ] Observe the existing syntax of the front matter block [^1] in the file.
- [ ]  Find the `leadership` variable and remove the following:
```
[Insert the lines of code to remove the specified leadership member]
```
- [ ] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [ ] [Insert name of project] page [^2]
- [ ] Once your pull request is merged, go to the initiating ER [^3]
  - [ ] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move initiating ER [^3] to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: [Insert project specific page URL here]
[^3]: initiating ER:  #[Insert ER number]
````
## Title
title of the issue that is created
```
Update Project Profile: [Project Name] Remove [Name of Volunteer]
```

**Comment by ExperimentsInHonesty** [comment]
# Instructions for filling out the Remove a leader template

1. Create a new issue using the [remove new leader to project](https://github.com/hackforla/website/issues/new?assignees=&labels=good+first+issue%2C+P-Feature%3A+Project+Info+and+Page%2C+role%3A+back+end%2FdevOps%2C+role%3A+front+end%2C+size%3A+0.25pt%2C+time+sensitive&projects=&template=project-profile-leader-updates---remove.md&title=Update+Project+Profile%3A+%5BName+of+Project%5D+Remove+%5BName+of+Volunteer%5D) template
2. Customize the title with name of project and person to add
3. Customize the body
    1.  In the ER, go to the value under the line "Project Card URL"
       1. Copy the file name 
       2. On the line that starts "- [ ] In your IDE…" replace brackets with the copy (the text we are expecting is a file name with the extension .md) 
    2. Go to https://github.com/hackforla/website/tree/gh-pages/_projects and find and click on the file
       1. click on code view
       2. copy the code block
       3. highlight the text on your new issue where it says `[Insert the lines of code to remove the specified leadership member]` and delete it and replace with the copy block
    3. On the checkbox line below, "- [ ] Verify the changes by viewing..." replace with name of project
      1. to Resource 3, find the project on the https://www.hackforla.org/projects/ page
          1. click on its link
          2. copy link
          3. replace brackets on resource 3 with the URL
5. Under "Relationships", add the originating ER as a parent
6.  Choose `Update comment` button
7. Add milestone `Onboarding flow`
8. Review, and if ready, add label
```
Ready for Prioritization
```
8. Add the URL to the ER
   1. if this is the first remove leader issue to be created, paste this into a comment and add the URL of the issue you just created.
```
### Removing Leader issues
-

### Complete
- [ ] Check this when all the removal issues have been made
```
   2. if there are already the block above in the issue, just add your new issue URL to the list. 
      1. Checking off the box, if it's the last one that needed to be created.

**Comment by ExperimentsInHonesty** [comment]
# Instructions for how to update project profile leadership requests from stakeholders

1. Review the Issue
2. Add the template to the top of the issue 
   1. customize the body project name
3. Move the issue to the `ER and Epics that are ready to make issues from` column
   1. If there are other issues that are time sensitive, and not project info issues, move this above them
   2. If there are other issues that are time sensitive and they also are project info issues, put this under them.

## template to add to the top of ERs that only have leadership changes
`````
### Overview
The [NAME OF PROJECT] project team needs its project leadership variable information updated, so that the current team members can be properly displayed on the Hack for LA website.

### Action Items
- [ ] Review the Project Update Data below and then follow the instructions for either adding or deleting team members from the project's .md

<details><summary>Adding new volunteers to team page</summary>

- [ ] Create a new issue using the [add new leader to project](https://github.com/hackforla/website/issues/new?assignees=&labels=good+first+issue%2C+P-Feature%3A+Project+Info+and+Page%2C+role%3A+back+end%2FdevOps%2C+role%3A+front+end%2C+size%3A+0.25pt%2C+time+sensitive&projects=&template=project-profile-leader-updates---add.md&title=Update+Project+Profile%3A+%5BProject+Name%5D+Add+%5BName+of+Volunteer%5D) template
- [ ]  Customize the title with name of project and person to add
- [ ]  Customize the body
   - [ ]   In the ER, go to the value under the line "Project Card URL"
      - [ ]  Copy the file name 
      - [ ] On the line that starts "- [ ] In your IDE…" replace brackets with the copy (the text we are expecting is a file name with the extension .md)
   - [ ] In the copy block below the line that starts, "Find the `leadership` variable and add the following profile."  add all the information from the ER for the person you are adding.
   - [ ] On the checkbox line below, "- [ ] Verify the changes by viewing..." replace with name of project
   - [ ] to Resource 3, find the project on the https://www.hackforla.org/projects/ page
      - [ ] click on its link
      - [ ] copy link
      - [ ] replace brackets on resource 3 with the URL
- [ ] Choose `Create` button
- [ ] Under Relationships, add this epic or ER as a parent to the issue you just created
- [ ] Add milestone `03.02 Onboarding flow`
- [ ] Review, and if ready, apply label `ready for merge team`
- [ ] At the bottom of this issue description, next to "Create sub-issue", click the drop-down menu arrow. Then, click "Add existing issue", search for the issue you just created, and select that issue to add as a sub-issue.
- [ ] Add the URL to a comment on this ER
    - [ ] if this is the first add leader issue to be created, paste this into a comment and add the URL of the issue you just created.
```
### Adding Leader issues
-

### Complete
- [ ] Check this when all the Add leader issues have been made
```
- [ ] if the above comment already exists, just add your new issue URL to the list. 
- [ ] Check off the box, if it's the last one that needed to be created.
</details> 

<details><summary>Removing volunteers from team page</summary>

- [ ] Create a new issue using the [remove leader from project](https://github.com/hackforla/website/issues/new?assignees=&labels=good+first+issue%2C+P-Feature%3A+Project+Info+and+Page%2C+role%3A+back+end%2FdevOps%2C+role%3A+front+end%2C+size%3A+0.25pt%2C+time+sensitive&projects=&template=project-profile-leader-updates---remove.md&title=Update+Project+Profile%3A+%5BName+of+Project%5D+Remove+%5BName+of+Volunteer%5D) template
- [ ] Customize the title with name of project and person to remove
- [ ] Customize the body
   - [ ] In the epic, go to the value under the line "Project Card URL"
      - [ ] Copy the file name 
      - [ ] On the line that starts "- [ ] In your IDE…" replace brackets with the copy (the text we are expecting is a file name with the extension .md) 
   - [ ] Go to https://github.com/hackforla/website/tree/gh-pages/_projects and find and click on the file
      - [ ] click on code view
      - [ ] copy the code block
      - [ ] highlight the text on your new issue where it says `[Insert the lines of code to remove the specified leadership member]` and delete it and replace with the copy block
   - [ ] On the checkbox line below, "- [ ] Verify the changes by viewing..." replace with name of project
   - [ ]  to Resource 2, find the project on the https://www.hackforla.org/projects/ page
      - [ ] click on its link
      - [ ] copy link
      - [ ] replace brackets on resource 2 with the URL
- [ ] Choose `Create` button
- [ ] Under Relationships, add this epic or ER as a parent to the issue you just created
- [ ] Add milestone `Onboarding flow`
- [ ] Review, and if ready, apply label `ready for merge team`
- [ ] At the bottom of this issue description, next to "Create sub-issue", click the drop-down menu arrow. Then, click "Add existing issue", search for the issue you just created, and select that issue to add as a sub-issue.
- [ ] Add the URL to a comment on this ER
   1. if this is the first remove leader issue to be created, paste this into a comment on this issue and add the URL of the issue you just created.
```
### Removing Leader issues
-

### Complete
- [ ] Check this when all the removal issues have been made
```
- [ ] if the above comment already exists, just add your new issue URL to the list. 
- [ ] Check off the box, if it's the last one that needed to be created.
</details> 

- [ ] When you have completed making all the add and remove issues
   - [ ] move this issue into the Questions/Review column
   - [ ] add the `ready for merge team` label

#### For Merge Team Members
- [ ] Check each sub-issue created for this ER/epic 
   - [ ] For each sub-issue that is ready, apply a `ready for prioritization` label
- [ ] If all sub-issues have been created so that the request in this ER/epic has been fulfilled, apply a `ready for product` label to this ER/epic

### Project Update Data 

--- 
`````

**Comment by ExperimentsInHonesty** [comment]
Make an issue to update the Project Profile Card review and update issue with the text from https://github.com/hackforla/website/issues/7478#issuecomment-2357066233 and a reference to this issue.

**Comment by HackforLABot** [comment]
@ExperimentsInHonesty, @Rabia2219, @essencegoff

Please add update using the below template (even if you have a pull request). Afterwards, remove the 'To Update !' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, October 1, 2024 at 12:04 AM PST.</sub>

**Comment by HackforLABot** [comment]
@ExperimentsInHonesty, @Rabia2219, @essencegoff

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, October 8, 2024 at 12:05 AM PST.</sub>

**Comment by ExperimentsInHonesty** [comment]
- Issue Made https://github.com/hackforla/website/issues/7596

**Comment by ExperimentsInHonesty** [comment]
- I created the new issue https://github.com/hackforla/website/issues/7596
